tests/boolean.rs

```diff
@@ -0,0 +1,127 @@
+#[cfg(test)]
+mod tests {
+    use yaiwr::{
+        ast::AstNode,
+        instruction::{Instruction, StackValue},
+        Calc,
+    };
+
+    #[test]
+    fn bool_literal_true_bc() {
```

Comment:
Random idea: if you provide a way to dump an AST to a textual format, then you could use lang_tester to make smaller tests.

(e.g. you could dump to json, and have it pretty printed)

Reply:
(or just use rust's `Debug` prints)

Reply:
I think I can use my logger in debug mode, cause I've seen that lang_tester support env config

Reply:
I'll do it in a separate PR.

Reply:
I was trying it out, and I found it quite cumbersome since this codebase is frequently changing, I prefer to keep bytecode tests as is instead of asserting on the textual format. Otherwise, every renaming or structural change in AST or Bytecode will force me to update many tests manually.
I will add this snapshot-testing when the YAIWR interpreter is more stable.